src/services/klantinteracties/types.ts

```diff
@@ -0,0 +1,88 @@
+import type { Contactverzoek } from "@/features/contact/contactverzoek/overzicht/types";
+
+export type DigitaalAdres = {
+  adres: string;
+  soortDigitaalAdres?: string;
+  omschrijving?: string;
+};
+
+export type BetrokkeneMetKlantContact = {
+  uuid: string;
+  wasPartij: { uuid: string; url: string };
+  klantContact: ExpandedKlantContactApiViewmodel;
+
+  digitaleAdressen: Array<{ uuid: string; url: string }>;
+  digitaleAdressenExpanded: Array<DigitaalAdres>;
+  contactnaam: {
+    achternaam: string;
+    voorletters: string;
+    voornaam: string;
+    voorvoegselAchternaam: string;
+  };
+};
+
+export type ExpandedKlantContactApiViewmodel = {
+  uuid: string;
+  url: string;
+  plaatsgevondenOp: string;
+  kanaal: string;
+  inhoud: string;
+  onderwerp: string;
+  hadBetrokkenActoren: Array<{
+    soortActor: string;
+    naam: string;
+    actorIdentificator: {
+      objectId: string;
+    };
+  }>;
+  internetaak: InternetaakApiViewModel;
+};
+
+export type InternetaakApiViewModel = {
+  uuid: string;
+  url: string;
+  nummer: string;
+  gevraagdeHandeling: string;
+  toegewezenAanActoren: Array<{
+    uuid: string;
+    url: string;
+  }>;
+  toelichting: string;
+  status: string;
+  toegewezenOp: string;
+  afgehandeldOp: string;
+  actor: ActorApiViewModel;
+};
+
+export type ActorApiViewModel = {
+  uuid: string;
+  url: string;
+  naam: string;
+  soortActor: string;
+  indicatieActief: boolean;
+};
+
+export interface MedewerkerIdentificatie {
+  identificatie: string;
+  achternaam: string;
+  voorletters: string;
+  voorvoegselAchternaam: string;
+}
+
+export interface ContactmomentViewModel {
```

Comment:
ivm met consistentie heb je hier bewust het api stukje in de viewmodel weggelaten?

Reply:
namen van types consistenter gemaakt